### DISCUSSION TITLE
如何从页面获取当前菜单信息？

### DISCUSSION THREAD

**Discussion by hyzx86** [umijs/umi#11021] [Q&A]
如题~

我的菜单是从数据库动态加载过来的，页面路由是一个动态路由
我在菜单上配置了一些扩展属性来标识加载不同页面数据
此时如何通过页面组件获取到 当前菜单定义呢？

**Comment by fz6m** [comment]
[`useAppData`](https://umijs.org/docs/api/api#useappdata) 、[`useSelectedRoutes`](https://umijs.org/docs/api/api#useselectedroutes) ，这里面有，自己找一下。
